src/main/resources/assets/secretroomsmod/lang/de_de.lang

```diff
@@ -0,0 +1,64 @@
+# creative tab
+itemGroup.secretroomsmod=Geheim-Raum
+
+# chat messages
+message.secretroomsmod.oneWay.towards=-- !!! Einwegblock auf "zu dir" eingestellt !!! --
+message.secretroomsmod.oneWay.away=-- !!! Einwegblock auf "von dir weg" eingestellt !!! --
```

Comment:
I'd rather have this more coherent; you wouldn't write it like this using quotations in German.